Makefile

```diff
@@ -445,8 +445,7 @@ release-manifests:
 	cp metadata.yaml $(RELEASE_DIR)/metadata.yaml
 
 .PHONY: release-changelog
-release-changelog: $(RELEASE_NOTES) check-release-tag check-previous-release-tag check-github-token $(RELEASE_DIR) ## Builds the changelog for a release
-	$(RELEASE_NOTES) --debug --org $(GH_ORG_NAME) --repo $(GH_REPO_NAME) --start-sha $(shell git rev-list -n 1 ${PREVIOUS_VERSION}) --end-sha $(shell git rev-list -n 1 ${RELEASE_TAG}) --output $(RELEASE_DIR)/CHANGELOG.md --go-template go-template:$(REPO_ROOT)/hack/changelog.tpl --dependencies=false --branch=$(BRANCH)
+release-changelog: gh api repos/$(GH_ORG_NAME)/$(GH_REPO_NAME)/releases/generate-notes -F tag_name=${RELEASE_TAG} --jq .body #Generates release notes using Github release notes. 
```

Comment:
I think a couple of changes are needed here:

- This target should rely on `$(GH)`
- We should use `$(GH)` for the command instead of `gh`
-  We need to be saving the output to `$(RELEASE_DIR)/CHANGELOG.md`

Reply:
Ideally we then need to combine this output with some predefined header/footer text

Reply:
For example we need to add the following as a header:

```markdown
# Release notes for Cluster API Provider AWS (CAPA) $RELEASE_VERSION

[Documentation](https://cluster-api-aws.sigs.k8s.io/)

# Changelog since $PREVIOUS_VERSION

```

And a footer of something like:

```markdown
**The image for this release is:**
<INSERT_IMAGE>
 
Thanks to all our contributors!
```

Reply:
I have added the mentioned changes here.